apps/frontend/src/app/5_pages/BorrowPage/hooks/useGetMinCollateralRatio.ts

```diff
@@ -0,0 +1,53 @@
+import { useMemo } from 'react';
+
+import { Decimal } from '@sovryn/utils';
+
+import { MINIMUM_COLLATERAL_RATIO_BORROWING_MAINTENANCE } from '../../../../constants/lending';
+import { rskClient } from '../../../../utils/clients';
+import { useGetLoanParamsSetupsQuery } from '../../../../utils/graphql/rsk/generated';
+import { decimalic } from '../../../../utils/math';
+
+export const useGetMinCollateralRatio = (
+  loanToken: string | undefined,
+): Decimal => {
```

Comment:
You can move `useGetTokenContract` inside this hook instead of having it called each time you need CR in the parent component

```suggestion
  token: SupportedTokens,
): Decimal => {
const loanToken = useGetTokenContract(token)?.address;
```